src/main/java/uk/gov/laa/ccms/caab/client/EbsApiClient.java

```diff
@@ -210,5 +254,135 @@ public Mono<UserDetails> getUsers(final Integer providerId) {
 
   }
 
+  /**
+   * Retrieves proceeding detail.
+   *
+   * @return A Mono containing the ProceedingDetail or an error handler if an error occurs.
+   */
+  public Mono<ProceedingDetail> getProceeding(String proceedingCode) {
```

Comment:
missing @param in javadoc
final on params

Reply:
done